contracts/sg721-nt/src/lib.rs

```diff
@@ -3,11 +3,10 @@ use cosmwasm_std::entry_point;
 
 pub mod contract;
 pub mod msg;
-use cw721_base::Extension;
 use sg721::InstantiateMsg;
-use sg_std::StargazeMsgWrapper;
+pub use sg721_base::Extension;
 
-pub type Cw721Base<'a> = cw721_base::Cw721Contract<'a, Extension, StargazeMsgWrapper>;
+pub type Sg721Base<'a> = sg721_base::Cw721Base<'a>;
```

Comment:
This is the tricky bit... may want to rename this in `sg721-base`. This type already includes the extension and msg wrapper which is nice.